### DISCUSSION TITLE
land use-crop production

### DISCUSSION THREAD

**Discussion by mxq517** [JGCRI/gcam-core#418] [Q&A]
biomassGrass and biomassTree will only start to be used in 2025. The land area of this part of the data before 2025 is 0. What does the yield data in the file ag_For_Past_bio_base_IRR_MGMT refer to and how to get it?
![1713318605119](https://github.com/JGCRI/gcam-core/assets/162215530/7e1baa09-7b33-4268-ab0a-8e74a9a580dc)

**Comment by pkyle** [comment]
The base-year yield assigned to bioenergy crops (which don't exist in the base year) is used by the model in two ways. First, the future yields are assigned as the base-year yields combined with future assumed `AgProdChange`. Second, in the base year the model calculates an imputed `profit-rate` of the technology, equal to (P-C)*Y, where P is the biomass price `$/GJ`, C stands for all costs (sum of `nonLandVariableCost, water_td_irr cost, N fertilizer cost`, indicated in `$/GJ`), and Y is the yield `GJ/m2`. So that returns $/m2. The profit-rate is used by the land allocator, along with the exogenous `ghost-unnormalized-share` (assigned in `land_input_4_IRR_MGMT.xml` and `land_input_5_IRR_MGMT.xml`) to calculate a `profit-scaler` for the biomass land leaf that is passed forward to all periods, and is part of the logit share equation for land share.
